NineChronicles.Headless/GraphTypes/ActionQuery.cs

```diff
@@ -120,6 +121,50 @@ public ActionQuery()
                     };
                     return Codec.Encode(action.PlainValue);
                 });
+            Field<ByteStringType>(
+                name: "transferAsset",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Description = "Address of sender.",
+                        Name = "sender",
+                    },
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Description = "Address of recipient.",
+                        Name = "recipient",
+                    },
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Description = "A string value to be transferred.",
+                        Name = "amount",
+                    },
+                    new QueryArgument<NonNullGraphType<CurrencyType>>
+                    {
+                        Description = "A currency value to be transferred.",
+                        Name = "currency",
+                    },
+                    new QueryArgument<StringGraphType>
+                    {
+                        Description = "A 80-max length string to note.",
+                        Name = "memo",
+                    }
+                ),
+                resolve: context =>
+                {
+                    var sender = context.GetArgument<Address>("sender");
+                    var recipient = context.GetArgument<Address>("recipient");
+                    Currency currency = context.GetArgument<CurrencyEnum>("currency") switch
+                    {
+                        CurrencyEnum.NCG => CurrencyType.NCG,
+                        CurrencyEnum.CRYSTAL => CurrencyType.CRYSTAL,
+                        _ => throw new ExecutionError($"Unsupported Currency type.")
```

Comment:
It seems not needed.

```suggestion
                        _ => throw new ExecutionError("Unsupported Currency type.")
```